lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -32,6 +32,16 @@
     description:
     - If set, then sets variable value to this
     type: str
+  mode:
+    description:
+    - C(persist) persists a global system variable to
+      the mysqld-auto.cnf option file in the data directory (the variable will survive service restarts).
+    - C(persist_only) behaves like C(persist) but without setting the global variable runtime value.
```

Comment:
Does idempotency work with `persist_only`?

Reply:
yep, it started to work (after your question)... :) thanks! Good question is often better than a PR itself...